dali/util/numpy.cc

```diff
@@ -214,5 +214,27 @@ size_t HeaderData::nbytes() const {
   return type_info ? type_info->size() * size() : 0_uz;
 }
 
+Tensor<CPUBackend> ReadTensor(InputStream *src) {
+  numpy::HeaderData header;
+  numpy::ParseHeader(header, src);
+  src->SeekRead(header.data_offset, SEEK_SET);
+
+  Tensor<CPUBackend> data;
+  data.Resize(header.shape, header.type());
+  Index ret = src->Read(static_cast<uint8_t*>(data.raw_mutable_data()), header.nbytes());
+  DALI_ENFORCE(ret == (Index)header.nbytes(), "Failed to read numpy file");
```

Comment:
```suggestion
  auto ret = src->Read(static_cast<uint8_t*>(data.raw_mutable_data()), header.nbytes());
  DALI_ENFORCE(ret == header.nbytes(), "Failed to read numpy file");
```
those are both `size_t`. No need to cast?

Reply:
Fixed, thank you